### PR TITLE
Tekst replace

### PR DESCRIPTION
**Pull request by dida-code**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
what's the relation with #4458 ?

**Comment by dida-code** [comment]
Sorry, this is my first time encountering all of this. I'm doing this as a hobby and trying to understand how to host my application on Flathub.

**Comment by hfiguiere** [comment]
that doesn't answer the question

**Comment by dida-code** [comment]
it's the same app, I guess I repeated the action twice

**Comment by hfiguiere** [comment]
then close one of the two

File: com.dida.TekstReplace.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.dida.TekstReplace",
```
Comment (hfiguiere) [inline com.dida.TekstReplace.json:2]: `com.dida` doesn't seem to exist

File: com.dida.TekstReplace.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
```
Comment (hfiguiere) [inline com.dida.TekstReplace.json:4]: this runtime is EOL. 44 is the current one

File: com.dida.TekstReplace.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "modules": [
+        {
+            "name": "tekstreplace",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/dida-code/com-dida-TekstReplace",
+                    "branch" : "main"
```
Comment (hfiguiere) [inline com.dida.TekstReplace.json:33]: never a branch
you need a commit or a tag.

File: com.dida.TekstReplace.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                }
+            ]
+        }
+    ],
+    "build-options": {
+        "env": {}
+    },
+    "app-icons": [
```
Comment (hfiguiere) [inline com.dida.TekstReplace.json:41]: what is this for?

File: com.dida.TekstReplace.json (hunk 1 above)
Comment (hfiguiere) [reply com.dida.TekstReplace.json:2]: just to clarify: the domain on which the app-id is based shall exist. But you didn't answer that question either.

It's a requirement.

File: com.dida.TekstReplace.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                    "type": "git",
+                    "url": "https://github.com/dida-code/com-dida-TekstReplace",
+                    "branch" : "main"
+                }
+            ]
+        }
+    ],
+    "build-options": {
+        "env": {}
+    },
```
Comment (hfiguiere) [inline com.dida.TekstReplace.json:40]:
```suggestion
```

**Comment by dida-code** [comment]
I'll have to give up for now, there are a lot of errors

**Pull request closed without merging**